internal/cmd/marketplace/get.go

```diff
@@ -0,0 +1,125 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package marketplace
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mia-platform/miactl/internal/client"
+	"github.com/mia-platform/miactl/internal/clioptions"
+	"github.com/spf13/cobra"
+)
+
+const (
+	getMarketplaceEndpoint        = "/api/backend/marketplace/%s"
+	JSON                   string = "json"
+	YAML                   string = "yaml"
+)
+
+// GetCmd return a new cobra command for getting a single marketplace resource
+func GetCmd(options *clioptions.CLIOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get [resource-id]",
```

Comment:
this id is supposed to be the _id of the item or a human readable id?

Reply:
the _id of the item that can be retrieved via `miactl marketplace list`